mate-base/caja/caja-1.27.2.ebuild

```diff
@@ -52,6 +52,11 @@ BDEPEND="${COMMON_DEPEND}
 	virtual/pkgconfig
 "
 
+RDEPEND="
```

Comment:
This should really be RDEPEND="${COMMON_DEPEND}", surely. And then you don't need the rest of it in there as they're already in COMMON_DEPEND.

Reply:
My thoughts here are it only needs those two packages as a RDEPEND so it's cleaner and I'd like to move 1.27 to that model ready for when it becomes 1.28 however `RDEPEND="${COMMON_DEPEND}"` will work as well

Reply:
I _really_ doubt it only needs those two as RDEPEND.

Reply:
I did have it reviewed by parona while learning how to use qa-vdb better than I have been however I'm happy to re-add for now.